content/blog/2021/09/managing-kamelets-with-kn/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Managing Kamelet event sources with kn"
+date: 2021-09-30
+draft: false
+authors: [christophd]
+categories: ["Kamelets", "Camel K"]
+preview: "How to use the Knative Kamelet event source plugin"
+---
+
+The latest community version of the [Knative client](https://github.com/knative/client) v0.26 includes a new kn plugin for managing Kamelets as Knative event sources (GitHub: [knative-sandbox/kamelet-plugin-source-kamelet](https://github.com/knative-sandbox/kn-plugin-source-kamelet)). With the new plugin users of the kn tooling can directly list the available Kamelet sources and bind these Kamelets to Knative resources such as brokers, channels or services.
+
+The Kamelets facilitate a whole new world of event source possibilities allowing users to connect to external services (AWS, Twitter, Telegram, Postgres) as part of Knative eventing.
+
+## Kamelets - what is this?
+
+Kamelets (Kamel route snippets) introduce a new concept in [Apache Camel](https://github.com/apache/camel) that allows users to connect to external systems via a simplified ready-to-use connectors, hiding all the low level details about how those connections are implemented.
+
+The user only provides a set of properties that the Kamelet needs for connecting to a foreign service (e.g. authorization token, target destination, connection credentials). The heavy lift of establishing the connection and exchanging data is done with the Camel components that run as part of the Kamelet runtime.
+
+If you want to use Kamelets on your K8s cluster simply install the Camel K operator ([installation guide](/camel-k/latest/installation/installation.html)).
+
+The user can then choose from a constantly growing [Kamelet catalog](/camel-kamelets/latest/index.html) that provides a great ensemble of ready-to-use Kamelets written by Apache Camel experts.
```

Comment:
Not sure if we should say that its written by Camel experts, as it may sound like that a Kamelet can only be written by such people. Where the goal is the opposite.

Reply:
np will remove that part